setup-readme.sh

```diff
@@ -152,6 +150,26 @@ technical issues needs to occur on the mailing list.
 
 You might also like to familiarize yourself with other
 [${group_type} documents](https://datatracker.ietf.org/${group_type_abbr}/${wg_all}/documents/).
+
+## How to Contribute
+
+Contributions can be made by creating pull requests, opening an issue, or
+posting to the working group mailing list. See above for the email address
+and a note about policy.
+
+There are several ways to create a pull request ("PR"):
```

Comment:
I would start by pointing to the GH docs.

I would not suggest emailing users, except as a backup.

Something like:

```markdown
[GitHub documentation](link-here) describes how you can make a pull request.
You can contribute using the GitHub UI as follows:

* list
* of
* stuff

Document authors/editors are often happy to accept contributions of text.
They might be willing to help you through the process.  Email them and ask.
```